pkg/ui/src/Header/Header.svelte

```diff
@@ -0,0 +1,93 @@
+<script lang="ts">
+	import { getContext, onMount } from 'svelte';
+
+	import { TABS, type TabContext } from './HeaderTabContext.svelte';
+	import HeaderTab from './HeaderTab.svelte';
+
+	export let title = 'Stock';
+	export let tabNames: string[];
+	export let initialTabIx = 0;
+	export let ariaLabel = 'Select a tab';
```

Comment:
When I said 
> It would have the nav baked in
I meant something like this:
```typescript
// Notice the 'const', not 'let'
const links = [
  {
    label: "Stock",
    href: "/inventory/stock"
  },
  {
    label: "Inbound",
    href: "/inventory/inbound"
  },
  {
    label: "Outbound",
    href: "/inventory/outbound"
  }
]
```